recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  skip: true  # [not (linux and py27)]
+  skip: true  # [not linux]
```

Comment:
This is why I was wondering about `py` selectors, @isuruf. Seems that in this case it resulted in the build being skipped entirely. So had to drop it. Admittedly an improvement for this recipe, but IDK how that carries over generally.